ragnarok_packets/src/lib.rs

```diff
@@ -1793,6 +1800,21 @@ pub struct ObjectiveDetails1 {
     pub mob_name: String,
 }
 
+#[derive(Debug, Clone, Packet, ServerPacket, MapServer)]
+#[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
+#[header(0x008A)]
+pub struct NotificationPacket {
```

Comment:
Where did you get the name `NotificationPacket`?

Reply:
I don't know what this packet is about, is a placeholder name.